cmd/gendeepcopy/deep_copy.go

```diff
@@ -92,7 +100,14 @@ func main() {
 			generator.OverwritePackage(vals[0], vals[1])
 		}
 	}
-	for _, knownType := range api.Scheme.KnownTypes(version) {
+	//handle the special cases of groupVersion = "api/" and "experimental/"
+	if version == "" {
```

Comment:
Shouldnt we be doing this only if group == "api"?
For ex: for groupVersion="users/", we do want to pass "users/" to api.Scheme.KnownTypes() right?